docs/content/preview/yugabyte-cloud/managed-security/managed-authentication.md

```diff
@@ -67,12 +70,65 @@ In addition, to configure Entra federated authentication in YugabyteDB Managed,
 
 Refer to [Create a new client secret](https://learn.microsoft.com/en-us/entra/identity-platform/howto-create-service-principal-portal#option-3-create-a-new-client-secret) in the Microsoft documentation.
 
-### Configure
+**Configure**
 
-To configure federated authentication, do the following:
+To configure federated authentication using Entra, do the following:
 
 1. Navigate to **Security > Access Control > Authentication**, then click **Enable Federated Authentication** to display the **Enable Federated Authentication** dialog.
+1. Choose the Microsoft Entra ID identity provider.
 1. Enter your Entra application client ID and secret.
 1. Click **Enable**.
 
 You are redirected to sign in to your IdP to test the connection. Once test connection is successful, federated authentication is enabled.
+
+  {{% /tab %}}
+
+  {{% tab header="Ping" lang="ping" %}}
+
+To use PingOne for your IdP, do the following:
+
+1. Sign in to your PingIdentity account and create an application.
+
+    - Under **Applications**, add a new application.
+    - Enter a name for the application.
+    - Set **Application Type** to OIDC Web App.
+    - Click **Save**.
+
+1. Select the application you created and, on the **Configuration** tab, make sure **Response Type** is set to Code and **Grant Type** to Authorization Code.
+
+1. Add <https://yugabyte-cloud.okta.com/oauth2/v1/authorize/callback> in **Redirect URIs**.
+
+1. Set **Token Endpoint Authentication Method** to Client Secret Post.
+
+1. Set **Initiate Login URI** to <https://cloud.yugabyte.com/login>.
+
+1. On the **Resources** tab, add openid, email, profile as ALLOWED SCOPES.
+
+1. Optionally, you can configure **Policies** and **Attribute Mappings** as required.
+
+1. On the **Access** tab, assign user groups to the application.
```

Comment:
```suggestion
1. On the **Access** tab, assign appropriate user groups to the application.
```

It's a minor nit, but since we're not defining the user groups they need to assign, thought this might be better